mewarpx/mewarpx/mepicmi.py

```diff
@@ -0,0 +1,40 @@
+"""
+Child classes of WarpX Picmi objects to support ME utility functions and
+extended functionality without going through their standards.
+"""
+import logging
+
+from pywarpx import picmi
+
+# Get module-level logger
+logger = logging.getLogger(__name__)
+
+
+class Species(picmi.Species):
```

Comment:
We should add a function to this class that will call `mwxrun.simulation.add_species` with the current species so that we will no longer need separate definitions of new particles and installation of particles calls in the run scripts.